File: Modul5/Python/PRAK503-2310817210003-MuhammadAzrianzan.py
def maksimal(a, b):
    if b > a:
        nilaiMaks = b
    else:
        nilaiMaks = a
    return nilaiMaks

def minimal(a, b):
    if b < a:
        nilaiMinim = b
    else:
        nilaiMinim = a
    return nilaiMinim

batas = 0
maks = -100000
minim = 100000
bilangan = int(input(""))

while batas < bilangan:
    nilai = int(input(""))
    maks = maksimal(maks, nilai)
    minim = minimal(minim, nilai)
    batas += 1
print(maks, minim)

File: Modul5/C/PRAK503-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int maksimal(int a, int b){
    int nilaiMaks;
    if (b > a) {
        nilaiMaks = b;
    }
    else {
        nilaiMaks = a;
    }
    return nilaiMaks;
}

int minimal(int a, int b){
    int nilaiMinim;
    if (b < a) {
        nilaiMinim = b;
    }
    else {
        nilaiMinim = a;
    }
    return nilaiMinim;
}

int main(){
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf("%d", &bilangan);

    while(batas < bilangan){
        int nilai;
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
    }
    printf("%d %d",maks,minim);
    return 0;
}
